### PR TITLE
Api/articles

### PR DESCRIPTION
**Pull request by omar-baggio**

### CODE REVIEW

File: model/articles.model.js
Comment (hevdevs) [inline model/articles.model.js:25]: Nice!

File: __tests__/app.test.js
Comment (hevdevs) [inline __tests__/app.test.js]: remove .only in test

**Review by hevdevs — CHANGES_REQUESTED** [review]
This is looking great so far Omar! 🎉 
We still need one more test for this endpoint though. We another 200 test to assert that the response by default is sorted by date in descending order.
Have a look at jest-sorted to do this: https://npmjs.com/package/jest-sorted
Please put in another PR when done

File: __tests__/app.test.js
Comment (corkesw) [inline __tests__/app.test.js]: This doesn't look quite right, you should take a look at the docs for jest-sorted here...

https://www.npmjs.com/package/jest-sorted?activeTab=readme

The example when checking the sort order of an array of objects:


`expect([{ id: 1 }, { id: 2 }, { id: 3 }]).toBeSortedBy('id');`

**Comment by corkesw** [comment]
Hey Omar, nearly there! Just take a look at the jest-sorted docs again to see which matcher should be used and how.

File: __tests__/app.test.js
Comment (hevdevs) [inline __tests__/app.test.js:117]: Great!

File: __tests__/app.test.js
Comment (hevdevs) [inline __tests__/app.test.js:119]: remove console.log before merge

**Review by hevdevs — APPROVED** [review]
All good to merge Omar 🚀🥳

**Pull request merged**
